charts/substra-backend/values.yaml

```diff
@@ -203,6 +203,7 @@ docker-registry:
 celerybeat:
   replicaCount: 1
   expiredTokensFlushPeriod: 86400
+  maximumImagesDuration: 86400
```

Comment:
See [this comment](https://github.com/SubstraFoundation/substra-backend/pull/399#discussion_r612190882)